### PR TITLE
Support multi-element qtrl sequences

### PR DESCRIPTION
**Pull request by singular-value**

### CODE REVIEW

File: cirq_superstaq/aqt.py [hunk 1]
```diff
@@ -14,34 +13,54 @@
...
+        circuits: Union[cirq.Circuit, List[cirq.Circuit]],
+        seq: Optional["qtrl.sequencer.Sequence"] = None,
+    ) -> None:
+        if isinstance(circuits, cirq.Circuit):
+            self.circuit = circuits
+        else:
+            self.circuits = circuits
+        self.seq = seq
 
+    def __repr__(self) -> str:
```
Comment (vtomole) [inline cirq_superstaq/aqt.py:28]: Let's do

```python
  def __repr__(self) -> str:
        if hasattr(self, "circuit"):
            return f"AQTCompilerOutput({self.circuit!r}, {self.seq})"
        return f"AQTCompilerOutput({self.circuits!r}, {self.seq})"
```

**Review by vtomole — CHANGES_REQUESTED** [review]
Small suggestion then LGTM

File: cirq_superstaq/aqt.py (hunk 1 above)
Comment (singular-value) [reply cirq_superstaq/aqt.py:28]: done

**Review by vtomole — APPROVED** [review]
LGTM

**Pull request merged**